### DISCUSSION TITLE
Use DaisyUI with template

### DISCUSSION THREAD

**Discussion by ojhawkins** [Knogobert/ntn-boilerplate#124] [General]
I have been trying to get DaisyUI working with this template very unsuccessfully.

I install and add **daisyui** plugin into the `tailwind.config.js` as per below.

`plugins: [require('daisyui')]`

With this alone, when I run dev I get 4 errors `Variant cannot be generated because selector contains no classes.`.

The most promising thing I can find relating to this problem is discussed here https://github.com/tailwindlabs/tailwindcss/issues/1833.

I have tired replacing the posscss plugins to match what is discussed as per below but still get the same issue.

```
postcss: {
  plugins: {
	'postcss-import': {},
	'postcss-nested': {},
	tailwindcss: {},
  },
}
```

This might be too specific of a question but thought I was ask none the less.

**Comment by ojhawkins** [comment]
Well I think I have solved my own problem, I upgraded tailwindcss to version 3 (@nuxtjs/tailwindcss >= 5).

It complies now so just need to see if there are any breaking changes.

**Reply by Knogobert** [reply]
Nice work, I'm working on updating to the latest packages so it should be fixed in the next bigger release soon. Thanks for coming back with your solution!
